pom.xml

```diff
@@ -170,6 +170,7 @@
                         <param>edu.ucsb.cs156.courses.config.SecurityConfig</param>
                         <param>edu.ucsb.cs156.courses.config.SecurityConfig.MyCsrfRequestMatcher</param>
                         <param>edu.ucsb.cs156.courses.config.SpringFoxConfig</param>
+                        <param>edu.ucsb.cs156.courses.documents.CoursePage</param>
```

Comment:
Why are we excluding CoursePage from mutation testing?

This doesn't sound right to me.

I think you need to remove this and do proper testing, unless you can make a solid case that it's not appropriate.

Show me what the mutation test report says when this is removed, and we can discuss it.

And a MINIMUM, please make an issue in your repo that asks the programmer to remove 
```
<param>edu.ucsb.cs156.courses.documents.CoursePage</param>
``` 

and implement appropriate tests.

Reply:
When i run it it shows this, it seems to be just line coverage.
<img width="746" alt="image" src="https://user-images.githubusercontent.com/77415714/171741754-ec030850-ddc3-4ad6-95ce-aeda4edc6ce4.png">
<img width="1356" alt="image" src="https://user-images.githubusercontent.com/77415714/171741785-d149fe56-c462-4df2-a931-d05a52eda600.png">